.travis.yml

```diff
@@ -65,7 +65,7 @@ jobs:
 
     - stage: baseline
       env: TOXENV=py27-pexpect,py27-trial
-    - env: TOXENV=py37-numpy-xdist
+    - env: TOXENV=py37-lsof-numpy-xdist PYTEST_COVERAGE=1
```

Comment:
Given that `lsof` is only used to cover the code for it (we do not react / handle its warnings anyway), this could also be done with a faster job for less overhead maybe.